README.md

```diff
@@ -227,6 +227,9 @@ The repository has the following permanent branches:
 
  * **develop** This contains the latest code. All the contributing PRs must be sent to this branch. When we want to release the next version of the app, this branch is merged into the `master` branch. This is the branch that is used in the deployed version of the app on Heroku.
 
+ * **bit** This branch is for ms-backend version specific to BridgeInTech project. All the contributing PRs related to BIT-MS integration issue must be sent to this branch.
```

Comment:
@The-Keshav-Agarwal add a link to BIT repo as well

Reply:
@vj-codes sir where should I add the link, at the end or after the first line or in between a word?

Reply:
@The-Keshav-Agarwal I meant hyperlinking the word BridgeInTech :)